src/platforms/minishift.ts

```diff
@@ -48,14 +48,6 @@ export class MinishiftHelper {
       },
       // { title: 'Verify minishift memory configuration', skip: () => 'Not implemented yet', task: () => {}},
       // { title: 'Verify kubernetes version', skip: () => 'Not implemented yet', task: () => {}},
-      { title: 'Retrieving minishift IP and domain for routes URLs',
```

Comment:
what if you use helm or minishift addon on minishift ?

Reply:
AFAIK minishift addon does not need domain as well.

About helm - I'm not even sure that it's a working case at all.
If we need it - we should adapt helm chart to be run onto OpenShift.
Like, use Routes instead of Ingresses, set OpenShift as Che Server infra because K8s infra uses namespaces instead of project and it's supposed to fail.

Reply:
BTW Chectl does not allow it 
![Screenshot_20190802_123403](https://user-images.githubusercontent.com/5887312/62360607-dabedb00-b521-11e9-8a99-6721a27b9c06.png)